test/fixtures/geoip-fixture/src/main/java/fixture/geoip/GeoIpHttpFixture.java

```diff
@@ -45,6 +50,38 @@
                 db.transferTo(outputStream);
             }
         });
+        this.server.createContext("/cli", exchange -> {
+            String fileName = exchange.getRequestURI().getPath().replaceAll(".*/cli/", "");
+            Path target = Path.of("target").resolve(fileName);
+            if (Files.isRegularFile(target)) {
+                try (OutputStream outputStream = exchange.getResponseBody();
+                     InputStream db = Files.newInputStream(target)) {
+                    exchange.sendResponseHeaders(200, 0);
+                    db.transferTo(outputStream);
+                } catch (Exception e) {
+                    exchange.sendResponseHeaders(500, 0);
+                    exchange.getResponseBody().close();
+                }
+            } else {
+                exchange.sendResponseHeaders(404, 0);
+                exchange.getResponseBody().close();
+            }
+        });
+    }
+
+    private void copyFiles() throws Exception {
+        Path source = Path.of("source");
+        Files.createDirectory(source);
+
+        Path target = Path.of("target");
```

Comment:
Do these paths end up in the build directory of the geoid-fixture module?
Otherwise perhaps we should create tmp dirs instead?

Reply:
Fixtures are run inside docker container which is then discarded, I think we can get away with using current directory